flixel/graphics/tile/FlxDrawQuadsItem.hx

```diff
@@ -12,7 +12,7 @@ import openfl.Vector;
 
 class FlxDrawQuadsItem extends FlxDrawBaseItem<FlxDrawQuadsItem>
 {
-	static inline var VERTICES_PER_QUAD = #if (openfl >= "8.5.0") 4 #else 6 #end;
+	static inline var VERTICES_PER_QUAD = #if (openfl >= "8.5.0") 1 #else 1 #end;
```

Comment:
?

Reply:
Sorry, I didn't mean to push that but accidentally did when I did the force ppush to fix the codeclimate issue. It's probably worth looking at first, though, tbh. I'm not sure the looping is necessary.